utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+
+import os
+import sys
+import pandas as pd
+import psycopg2
+from psycopg2.extras import RealDictCursor
+from psycopg2 import sql
+from sqlalchemy import create_engine
+from typing import List, Optional
+import logging
+import json
+def serialize_row(row):
```

Ruff (0.12.2)

Missing return type annotation for public function `serialize_row`

(ANN201)